docs/blog/2021-community-kickoff.rst

```diff
@@ -0,0 +1,186 @@
+As we noted in our `ticket choices`_ page for Portland,
+we had a lot of outstanding debts and we didn't know how much support we would get with our online conferences.
+It was easy to see a worst case scenario where our venues didn't refund us and people didn't support the online events.
+
+We are happy to report that we received overwhelming support from our vendors, sponsors, speakers, team, and attendees:
+
+* Most of our vendors refunded our down payments in full. Our A/V and captioning folks adapted perfectly to the new format.
+* Only 1 of our sponsors canceled because of the transition to a virtual event.
+* Our speakers pulled together to record videos and handle live online Q&A.
+* Our team pulled together to run virtual events that exceeded our expectations for what those events could be.
+* Our attendees, most importantly, made our virtual events *feel* like our conferences, with a supportive and generous spirit.
+
+2020 has been a hard year,
+but we wanted to take this chance to give thanks to those who made 2020 a special year.
+We feel incredibly lucky to be able to support this wonderful community,
+and we look forward to making 2021 the best year yet,
+with your help.
+
+.. _ticket choices: https://www.writethedocs.org/conf/portland/2020/ticket-choices/
+
+Looking forward
+---------------
+
+One of the largest things we noticed in 2020 was how much more support our community needed.
+As we scrambled to move our events online,
+people were also adjusting to working remotely for the first time.
+We saw lots of activity in our Slack network,
+as people were trying to connect online.
+Meetups slowly adjusted,
+trying to move online and keep some momentum going.
+
+Write the Docs itself was able to provide similar value as in the past,
+but we didn't have the resources to adjust to the changing world.
+The profits from our conferences pay for everything we do,
+which constrains our ability to adjust to events as they happen.
+
+We are planning a way for our community to support the organization directly.
+We are proud that all the content that Write the Docs produces is free to everyone,
+and we don't plan to change that any time soon.
+The goal for 2021 is to be able to provide additional support to our community,
+paid for by members in that community.
+
+We're still working out the details,
+and will be posting a `WEP`_ soon to ask the community for feedback.
+We hope that you'll support us in building a community that is better able to support you.
+
+.. _WEP: https://www.writethedocs.org/blog/introducing-weps/
+
+Write the Docs Quorum Meetups
+-----------------------------
+
+One of the first things we're excited about this year is an evolution of our meetups: Quorum meetups.
+These quarterly, remote, "super" meetups are an hour long and include a presentation by a speaker followed by breakout networking.
+
+Write the Docs is running two Quorum pilot programs for 2021:
```

Comment:
```suggestion
Write the Docs is running three Quorum pilot programs for 2021:
```
I know that only two are specifically planned, but "two" and then three bullets feels weird?